moveos/moveos-client/src/client.rs

```diff
@@ -0,0 +1,178 @@
+struct SendTxRequest {
+    sender_address: String,
+    public_key: Vec<u8>,
+    program: Vec<u8>,
+    sequencer_number: u64,
+    expiration_time: u64,
+    signature: Vec<u8>,
+}
+
+struct SendTxnResponse {
+    // The order of a user's transactions in a block.
+    order: u128,
```

Comment:
this is the global order or the order in a block? we do not have the block concept.

Reply:
user's transaction's order: the execution order will follow this order strictly. e.g. txn with y order must be no later than txn with x order, if x > y @steelgeek091 

that could be comment of order